_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -0,0 +1,192 @@
+---
+title: 'Bitcoin Optech Newsletter #108'
+permalink: /en/newsletters/2020/07/29/
+name: 2020-07-29-newsletter
+slug: 2020-07-29-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to allow upgrading LN
+channel commitment transaction formats without opening new channels and includes
+a field report from River Financial about building wallet software using PSBTs
+and descriptors. Also included are our regular sections with selected questions and
+answers from the Bitcoin StackExchange, recent releases and release
+candidates, and notable changes to popular Bitcoin infrastructure
+projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Upgrading channel commitment formats:** Olaoluwa Osuntokun
+  [posted][osuntokun upgrade] to the Lightning-Dev mailing list this
+  week with a suggested extension to the LN specification that will
```

Comment:
µ-nits in this paragraph:

- s/will/would/ allow  (there is also "Osuntokun’s proposal would allow" and "would port" in the same paragraph)

- "between them" seems unneeded

- maybe s/upgrade to new/change/ commitment formats

- s/on chain/on-chain/? ("put something on chain" seems awkward)

(started reviewing, but I'll be offline for half an hour, so posting this in case GH loses it)